src/main/java/com/scriptbasic/syntax/commands/AbstractCommandAnalyzer.java

```diff
@@ -93,17 +93,16 @@ protected boolean isKeyWord(final String keyword) throws AnalysisException {
     }
 
     /**
-     * Checks that there are no extra characters on a program line when the line
-     * analyzer thinks that it has finished analyzing the line. If there are
+     * Checks that there are no extra characters on when the line
```

Comment:
"on when"

Please do not just delete some words. Ensure that the remaining sentence makes sense and is correct.